docs/v0.10/Readme.md

```diff
@@ -461,6 +462,26 @@ This is only valid if you have set your image to be resized. If you specify your
 **thumbnails**
 Thumbnails takes an array of objects. Each object is a new thumbnail that is created. Each object contains 2 values, the `name` and `scale` percentage. The `name` will be attached to your thumbnail image (as an example say the image you uploaded was ABC.jpg a thumbnail with the `name` of `medium` would now be created at ABC-medium.jpg). The `scale` is the percentage amount you want that thumbnail to scale. This value will be a percentage of the *resize* width and height if specified.
 
+#### Date
+
+```js
+{
+    "format" : "Y-m-d"
+}
+```
+
+The date input field is where you can input a date. In the JSON above you can specify the `format` value of the output of the date.
```

Comment:
The date input field is where you can input a date. In the JSON above you can specify the `format` value of the output of the date. It allows you to display a formatted `date` in _browse_ and _read_ views, using Carbon's `formatLocalized()` method